modules/apps/app-builder/app-builder-web/src/main/resources/META-INF/resources/js/components/list-view/ListView.es.js

```diff
@@ -113,7 +113,7 @@ export default withRouter(
 		}
 
 		const selectedFilters = getSelectedFilters(filters, query.filters);
-		const isEmpty = totalCount === 0;
+		const isEmpty = totalCount === 0 || items.length === 0;
```

Comment:
Hey @albinojunior,
this fix seems a little strange to me. The backend must return totalCount as 0 when there aren't standard apps. Please call me on slack to discuss about it.

 Thx man

Reply:
sent backend fix here https://github.com/liferay-workflow/liferay-portal/pull/29